apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-description/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * WordPress dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import './style.scss';
+
+registerBlockType( 'a8c/site-description', {
+	title: __( 'Site Description2' ),
```

Comment:
@kwight Typo?

Reply:
That was me trying to me make sure I was getting the latest build on my local environment 😅. Will prep a PR reverting this in a bit

Reply:
https://github.com/Automattic/wp-calypso/pull/34096